src/pydrex/utils.py

```diff
@@ -0,0 +1,13 @@
+"""> PyDRex: Miscellaneous utility methods."""
+import numpy as np
+
+
+def skip_nans(a):
+    """Skip NaN values in array."""
```

Comment:
Skip or remove?

Reply:
Well, Julia has `skipmissing` which removes `missing` values :joy: I'll rename it though.